src/configure/configure.ts

```diff
@@ -101,12 +102,16 @@ class PipelineConfigurer {
 
         });
 
+        telemetryHelper.setCurrentStep('PostPipelineCreation');
+        // This step should be determined by the resoruce target provider (azure app service, function app, aks) type and pipelineProvider(azure pipeline vs github)
+        await this.updateScmType(queuedPipeline);
```

Comment:
do we need to await for update?

Reply:
Not necessarily, I'll remove await for updatescm, updatemetadata and sendDeploymentLog